tests/test_sample_rayservice_yamls.py

```diff
@@ -62,7 +62,14 @@ def wait(self):
                 namespace = self.namespace, label_selector='ray.io/node-type=worker')
             serve_services = k8s_v1_api.list_namespaced_service(
                 namespace = self.namespace, label_selector =
-                f"ray.io/originated-from=RayService_{self.custom_resource_object['metadata']['name']}-serve")
+                f"ray.io/originated-from-cr-name={self.custom_resource_object['metadata']['name']},"
+                f"ray.io/originated-from-crd=RayService,"
+                f"ray.io/serve={self.custom_resource_object['metadata']['name']}-serve")
```

Comment:
TODO: The definition of `ray.io/serve` is so messy. It is possible to be `True/False` and `$SVC_NAME-serve`.